science/2022-03-03-science-blog-5/index_de.md

```diff
@@ -0,0 +1,898 @@
+</td>
+</tr>
+</tbody>
+<caption>
+Tab. 3: Kennzahlen zu Downloads und Nutzung (Apple)
+</caption>
+</table>
+
+Die folgende Abbildung enthält die wesentlichen Kennzahlen zum Download und Nutzungsverhalten.
+
+<figure>
+    <img src="./apple_installs_overview_plotly_de.png" class="figure-img img-fluid" width=100% alt="Angaben zur Installation (Apple)."/>
+    <figcaption aria-hidden="true">Abb. 17: Angaben zur Installation (Apple).</figcaption>
+</figure>
+
+Die Anzahl der monatlich aktiven Nutzenden kann auf 11,6 Mio. hochgerechnet werden (Stand: 27. Februar 2022). Hierbei wurde die Opt-In-Rate (die Rate der Nutzenden, die zugestimmt haben, App-Entwicklern ihre Diagnose- und Nutzungsinformationen zur Verfügung zu stellen) von 23% benutzt, deren Schwankung um diesen Mittelwert im vergangenen Jahr sehr gering war.
+
+Weiterhin ist zu beachten, dass *aktive* Geräte hier in etwa die Bedeutung der *aktiven Geräte pro Tag* von Google hat und wie dort etwa bei einem Drittel der monatlich aktiven Nutzenden liegt.
+
+Zunächst verwunderlich erscheint die Angabe von Downloads auf einen Desktop, wo doch ein solches Endgerät sicherlich nicht für das Proximity-Tracing benutzt werden kann. Die Erklärung hierfür ist die Möglichkeit, die Corona-Warn-App auch über den iTunes-Store zu beziehen. Dieses Verfahren wird insbesondere von Unternehmen und Instituten benutzt, um die Anwendung vielen Nutzenden komfortabel zuzuordnen.
+
+## 7 Vergleich der Angaben zu aktiven Nutzenden
+
+Für die Frage nach den aktiven Nutzenden (im erweiterten Sinne) scheint die Anzahl der monatlich aktiven Nutzenden gut geeignet. Wir geben in der folgenden Abbildung die Summe der monatlich aktiven Nutzenden (MAU) aus dem [Google-Play-Store](#61-google-play-store) und dem [Apple-App-Store](#62-apple-app-store) zusammen mit den Schätzungen aus der [Anzahl der Warnenden](#41-aus-der-anzahl-der-warnenden), der [CWA-Datenspende](#42-aus-der-cwa-datenspende) und der [Heuristik](#43-aus-einer-heuristik) an.
+
+<figure>
+    <img src="./google_estimates_plotly_de.png" class="figure-img img-fluid" width=100% alt="Verschiedene Schätzungen von aktiven Nutzenden (im erweiterten Sinne)."/>
+    <figcaption aria-hidden="true">Abb. 18: Verschiedene Schätzungen von aktiven Nutzenden (im erweiterten Sinne).</figcaption>
+</figure>
+
+Die [Schätzung aus der CWA-Datenspende (PPA)](#42-aus-der-cwa-datenspende) (aktuell 1; Stand: 28. Februar 2022) schwankt deutlich und ist damit nur mit großer Vorsicht zu genießen. Hier wurde aus dem Anteil der Datenspendenden auf die aktiven Nutzenden geschlossen. Aktuell ergeben sich 31 Mio. aktive Nutzende (Stand: 27. Februar 2022, [KI](https://de.wikipedia.org/wiki/Konfidenzintervall): 25,8 Mio. – 33,6 Mio.).
```

Comment:
"aktuell 1;" does not make sense, IHMO.

Reply:
Hi Tim,

you are totally right. Makes no sense. This should be "aktuell 16.4 Mio. spendende Endgeräte" at this point. Thank you very much!

Best regards
Tobias

Reply:
Thanks Tobias! I will provide a PR fixing this! 

Enjoy Friday and best regards
Tim

Reply:
PR https://github.com/corona-warn-app/cwa-website/pull/2506 created.

Reply:
ok, thank @Ein-Tim